src/components/ToDoContainer/index.js

```diff
@@ -1,9 +1,143 @@
 import React, { Component } from "react";
 
+
+let id = 0;
+
+class Add extends Component {
+  constructor(props) {
+    super(props); 
+    this.state = {value : ''};
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
```

Comment:
If you declare these methods using arrow notation, you won't have to `.bind()` these methods in the constructor:
```
handleChange = (event) => { ... }
```
In general, you should definitely review features of ES6 syntax such as this arrow notation!